seed/views/00302_ldu_case_overview.js

```diff
@@ -1,26 +1,33 @@
 exports.seed = function (knex, promise) {
-  var sql = `CREATE VIEW app.ldu_case_overview
+  var view = `CREATE VIEW app.ldu_case_overview
   WITH SCHEMABINDING
   AS
   SELECT
-      MAX(t.description) AS name
-    , MAX(tv.grade_code) AS grade_code
-    , SUM(tv.total_cases) AS total_cases
-    , SUM(tv.available_points) AS available_points
-    , SUM(tv.total_points) AS total_points
-    , SUM(tv.contracted_hours) AS contracted_hours
-    , SUM(tv.reduction_hours) AS reduction_hours
-    , MAX(tv.default_contracted_hours_po) AS default_contracted_hours_po
-    , MAX(tv.default_contracted_hours_pso) AS default_contracted_hours_pso
-    , MAX(l.id) AS id
-    , MAX(t.id) AS link_id
-  FROM app.team_case_overview tv
-    JOIN app.team t ON t.id = tv.id
+      t.description AS name
+    , SUM(w.total_cases) AS total_cases
+    , SUM(wpc.available_points) AS available_points
+    , SUM(wpc.total_points) AS total_points
+    , SUM(wpc.contracted_hours) AS contracted_hours
+    , SUM(wpc.reduction_hours) AS reduction_hours
+    , l.id AS id
+    , t.id AS link_id
+    , COUNT_BIG(*) AS count
```

Comment:
Why are we doing the count, is this used anywhere?

Reply:
Good question, it's a requirement for indexed views in MS SQL. https://docs.microsoft.com/en-us/sql/relational-databases/views/create-indexed-views.

```
If GROUP BY is present, the VIEW definition must contain COUNT_BIG(*) and must not contain HAVING. These GROUP BY restrictions are applicable only to the indexed view definition. A query can use an indexed view in its execution plan even if it does not satisfy these GROUP BY restrictions. 
```